d2app/app.go

```diff
@@ -601,3 +603,39 @@ func updateInitError(target d2interface.Surface) error {
 
 	return nil
 }
+
+func (a *App) ToMainMenu() {
```

Comment:
Can we turn this into a TransitionScene(d2enum.SceneType) or something?

Reply:
>I decided against a sceneManager.SetNextScene(d2enum.SceneMainMenu) method for a couple reasons. With explicit methods, we can keep the method smaller and easier to understand. Secondly, we can define what parameters are needed for the specific action.

Reply:
Some of the transitions may require a `filePath` or `connectionHost` or `connectionType` or a myriad of other arguments. This would be difficult to pass these cleanly in a generic transition interface.